main.py

```diff
@@ -258,6 +281,7 @@ def evaluate_genomes(genomes, config):
                 food_distances = list(
                     get_nearest_players_distances(player, food_list).values()
                 )[:FOOD_DETECTION]
+                # Not currently used

```

Comment:
delete it then you melon

Reply:
Didn't know if someone wanted to use it on a rainy day